### ISSUE TITLE
Fresh pull of repo won't successfully build cloud run example

### ISSUE THREAD

**Issue by joeholley** [GoogleCloudPlatform/cpp-samples#134]
Pulling the repo fresh and following the instructions in the readme results in this failure:

```
joeholley@instance-1:~/repos/git/cpp-samples/cloud-run-hello-world$ bash -x ./bootstrap-cloud-run-hello.sh
+ set -eu
+ [[ -z buildtest01 ]]
+ readonly GOOGLE_CLOUD_PROJECT=buildtest01
+ GOOGLE_CLOUD_PROJECT=buildtest01
+ readonly GOOGLE_CLOUD_REGION=asia-northeast1
+ GOOGLE_CLOUD_REGION=asia-northeast1
+ gcloud services enable cloudbuild.googleapis.com --project=buildtest01
+ gcloud services enable containerregistry.googleapis.com --project=buildtest01
+ gcloud services enable run.googleapis.com --project=buildtest01
++ git rev-parse --short HEAD
+ gcloud builds submit --project=buildtest01 --substitutions=SHORT_SHA=4433ef3 --config=cloudbuild.yaml
Creating temporary tarball archive of 8 file(s) totalling 12.9 KiB before compression.
Some files were not included in the source upload.

Check the gcloud log [/home/joeholley/.config/gcloud/logs/2021.05.27/07.05.46.811772.log] to see which files and the contents of the
default gcloudignore file used (see `$ gcloud topic gcloudignore` to learn
more).

Uploading tarball of [.] to [gs://buildtest01_cloudbuild/source/1622099146.878179-6db645fdfa974f41b0813c21348de0ab.tgz]
ERROR: (gcloud.builds.submit) INVALID_ARGUMENT: generic::invalid_argument: key "SHORT_SHA" in the substitution data is not matched in the template
```

Removing the substitution argument from the bootstrap script resolves the problem (I didn't see anywhere obvious in the cloudbuild.yaml where the SHORT_SHA substitution was needed and missing):
```
joeholley@instance-1:~/repos/git/cpp-samples/cloud-run-hello-world$ git diff
diff --git a/cloud-run-hello-world/bootstrap-cloud-run-hello.sh b/cloud-run-hello-world/bootstrap-cloud-run-hello.sh
index 1440f28..fe30357 100755
--- a/cloud-run-hello-world/bootstrap-cloud-run-hello.sh
+++ b/cloud-run-hello-world/bootstrap-cloud-run-hello.sh
@@ -34,7 +34,6 @@ gcloud services enable run.googleapis.com \
 # Build the Docker Images
 gcloud builds submit \
     "--project=${GOOGLE_CLOUD_PROJECT}" \
-    "--substitutions=SHORT_SHA=$(git rev-parse --short HEAD)" \
     "--config=cloudbuild.yaml"

 # Create a service account that will update the index
```

Let me know if you want me to submit a PR.

**Comment by coryan** [comment]
Thanks for the bug report, yes a PR would be very useful, thanks!

**Comment by coryan** [comment]
Fixed in #135
